### ISSUE TITLE
Better error message when missing .osrm.edge_penalties file

### ISSUE THREAD

**Issue by karenzshea** [Project-OSRM/osrm-backend#3114]
### Issue

In order to run contraction with a turn penalty file or speed segment file (`--turn-penalty-file` and `--speed-segment-file` flags) extraction must be run with the `--generate-edge-lookups` flag first. It is documented somewhere in the wiki, but not evident when using the binaries.

This is the error thrown when contraction with penalty/speed files are provided, but when extraction was run without `--generate-edge-lookups`

```
[info] Input file: monaco.osrm
[info] Threads: 4
[info] Loading edge-expanded graph representation
[info] Opening ../test/data/monaco.osrm.ebg
terminate called after throwing an instance of 'boost::interprocess::interprocess_exception'
  what():  No such file or directory
Aborted (core dumped)
```

This reads like `.osrm.ebg` is the missing file, but there's no indication that `.osrm.edge_penalties` is the file that's actually missing.
### To do
- return an error message like `Error: latest.osrm.edge_penalties file not found! Was extraction run with ---generate-edge-lookup?` when contraction attempts to open a missing `.osrm.edge_penalties` file
- Update the help/man info for `osrm-contract` to include a note with the `--speed-segment-file` and `--turn-penalty-file` options that `--generate-edge-lookup` _must_ be included during extraction.

Related: https://github.com/Project-OSRM/osrm-backend/issues/3047 and https://github.com/Project-OSRM/osrm-backend/issues/3046

**Comment by james-gould** [comment]
I'll fix this this afternoon.

**Comment by danpat** [comment]
With https://github.com/Project-OSRM/osrm-backend/pull/3321, at least the file reading side of this should be addressed.

**Comment by danpat** [comment]
Fixed by #3419
